### PR TITLE
ENH add py27 command

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Pushed the branch to main repo
* [x] CI passed on the branch

### CODE REVIEW

**Comment by scopatz** [comment]
It would be kinda cool if this were able to add any old version of Python, eg it also worked for 3.5. I don't think it would be a big modification to the PR.

**Comment by beckermr** [comment]
@scopatz good to merge?

**Comment by beckermr** [comment]
bump here @scopatz

**Comment by scopatz** [comment]
Thanks!

**Comment by isuruf** [comment]
You need to add `c_compiler` and `vc` entries and the `zip_keys` entry.

**Comment by beckermr** [comment]
For windows sure, but we are deprecating vs2008 and we have trouble with those on azure, so I decided to punt.

File: conda_forge_webservices/commands.py [hunk 1]
```diff
@@ -360,16 +401,64 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
...
+
+    # need to apply the selector - being very lazy here
+    with open(cbc_pth, "r") as fp:
+        lines = fp.readlines()
+    with open(cbc_pth, "w") as fp:
+        for line in lines:
+            if "pystr" in line:
+                line = line.replace(
+                    "pystr",
+                    "%s  # [not (aarch64 or ppc64le)]]" % pystr
```
Comment (isuruf) [inline conda_forge_webservices/commands.py]: @beckermr, then you should add `win` here too

Comment (beckermr) [reply conda_forge_webservices/commands.py]: +1 will do

File: conda_forge_webservices/commands.py [hunk 2]
```diff
@@ -306,6 +312,41 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
...
+                    extra_msg = "\n\nMerge this PR to enable Python %s.\n" % pyver
+
+                    if pyver == "2.7":
+                        extra_msg += (
+                            "WARNING: Python {pyver} reached "
+                            "end-of-life on 2020-01-01. "
+                            "`conda-forge` provides no support for Python {pyver} "
+                            "builds and "
+                            "all builds are provided on an "
+                            "\"as-is\" basis.\n".format(pyver=pyver)
```
Comment (isuruf) [inline conda_forge_webservices/commands.py]: Can you also add that `win` builds here are deprecated as vs2008 is not built anymore?

Comment (beckermr) [reply conda_forge_webservices/commands.py]: Ahh yeah. Will make another pr.

**Pull request merged**